### PR TITLE
Changes for (Add IsParseError() documentation #578)

### PR DESCRIPTION
**Pull request by karimalzalek**
Removed the section mentioning panic as behaviour is changed, and added a short error handling paragraph to explain errParse and IsPareseErr helper.

### CODE REVIEW

**Review by SoulPancake — COMMENTED** [review]
LG

File: README.md [hunk 1]
```diff
@@ -435,7 +435,17 @@ n, resp, err := client.Do(ctx, cmd).AsFtSearch()
...
 While the command builder is developer-friendly, the response parser is a little unfriendly. Developers must know what
-type of Redis response will be returned from the server beforehand and which parser they should use. Otherwise, it panics. 
+type of Redis response will be returned from the server beforehand and which parser they should use.
+
+Error Handling:
+If an incorrect parser function is chosen, an errParse will be returned. You can use the IsParseErr helper to check for this case.
+
+```golang
+// Attempt to parse the response. If a parsing error occurs, check if the error is a parse error and handle it.
+if err := client.Do(ctx, client.B().Get().Key("k").Build()).AsInt64(); IsParseErr(err) {
```
Comment (rueian) [inline README.md:445]:
``` suggestion
if err := client.Do(ctx, client.B().Get().Key("k").Build()).ToArray(); IsParseErr(err) {
```
Hi @karimalzalek, thank you for fixing the document but your example, `GET+AsInt64`, will not return an errParse. Maybe using something like `ToArray` as an example will be less confusing.

Also, could you help us emphasize that a user should normally fix the code by choosing the correct parser function, not using `IsParseErr`, in the comment?

**Comment by karimalzalek** [comment]
Hello @rueian , I changed the example to `ToArray()`,  and emphasized that the user needs to specify the correct parser beforehand. Let me know if the changes make sense.

**Comment by rueian** [comment]
Hi @karimalzalek, the PR looks good to me but please remove the `.vscode/settings.json`.

**Comment by karimalzalek** [comment]
@rueian yup, I removed the wrong config file, not sure how that made it's way through 😂

File: README.md [hunk 2]
```diff
@@ -434,8 +434,19 @@ n, resp, err := client.Do(ctx, cmd).AsFtSearch()
...
-While the command builder is developer-friendly, the response parser is a little unfriendly. Developers must know what
-type of Redis response will be returned from the server beforehand and which parser they should use. Otherwise, it panics. 
+While the command builder is developer-friendly, the response parser is a little unfriendly. Developers must know what type of Redis response will be returned from the server beforehand and which parser they should use.
+
+Error Handling:
+If an incorrect parser function is chosen, an errParse will be returned. Here's an example using ToArray which demonstrates this scenario:
+
+```golang
+// Attempt to parse the response. If a parsing error occurs, check if the error is a parse error and handle it.
+// Normally, you should fix the code by choosing the correct parser function
```
Comment (SoulPancake) [inline README.md:444]:
```suggestion
// Normally, you should fix the code by choosing the correct parser function.
```

**Comment by rueian** [comment]
Thanks @karimalzalek!

**Pull request merged**